stories/Image.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Image from '../src/components/Image/Image';
+
+
+storiesOf('Image', module).add('image', () => (
+  <Image />
+));
+
+storiesOf('Image', module).add('image click', () => (
```

Comment:
Should this change the way the image looks? If not, it shouldn't be in the storybook.